### PR TITLE
fix(studio): overlapping between search bar & node props side panel

### PR DESCRIPTION
**Pull request by sadda11asm**
Added another CSS style and choose that one whenever the side panel is open

### CODE REVIEW

**Comment by allardy** [comment]
Can you include a screenshot of before/after so the change is easier to see? Thanks!

**Comment by EFF** [comment]
The PR description isn't clear. Next time try to add more details and as @allardy said when it touches UI try to add before/after screencaps

**Comment by sadda11asm** [comment]
oops sorry guys, missed your replies, will do your requests asap

**Comment by sadda11asm** [comment]
**How it was**:
1) the search tab before nodePropsTab is opened
image
2) the search tab after nodePropsTab is opened (it is behind it)
image
**How it now**:
2) the search tab after nodePropsTab is opened 
image
When nodePropsTab is bigger:
image
@allardy @laurentlp @EFF here are the screenshots :)

**Comment by allardy** [comment]
I'm sure there's a way to implement this using CSS instead of adding another library and passing around the width of an opened panel. Can you look into that ?

**Comment by sadda11asm** [comment]
> I'm sure there's a way to implement this using CSS instead of adding another library and passing around the width of an opened panel. Can you look into that ?

Okay I will look into that

**Comment by sadda11asm** [comment]
@allardy I made changes per your request! Could you please review it again?

File: packages/studio-ui/src/web/views/FlowBuilder/index.tsx [hunk 1]
```diff
@@ -182,23 +182,27 @@ const FlowBuilder = (props: Props) => {
...
+      <div className={style.container}>
+        <div className={style.diagram}>
+          <Diagram
+            readOnly={readOnly}
+            showSearch={showSearch}
+            hideSearch={() => setShowSearch(false)}
+            handleFilterChanged={handleFilterChanged}
+            highlightFilter={highlightFilter}
+            mutexInfo={mutex}
+            showFlowNodeProps={props.showFlowNodeProps}
```
Comment (EFF) [inline packages/studio-ui/src/web/views/FlowBuilder/index.tsx:194]: This property is new ? how does that affect the ui ?

File: packages/ui-shared/src/MainLayout/Toolbar/index.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ const Toolbar: FC<ToolbarProps> = props => {
         <Tabs currentTab={props.currentTab} tabChange={props.tabChange} shouldFloat tabs={props.tabs} />
       )}
       {!!props.buttons?.length && (
-        <NavbarGroup className={cx(style.buttons, 'toolbar-buttons')} align={Alignment.RIGHT}>
+        <NavbarGroup style={{ marginTop: '5px' }} align={Alignment.RIGHT}>
```
Comment (laurentlp) [inline packages/ui-shared/src/MainLayout/Toolbar/index.tsx:20]: Move this into a CSS class.

File: packages/studio-ui/src/web/views/FlowBuilder/index.tsx (hunk 1 above)
Comment (laurentlp) [reply packages/studio-ui/src/web/views/FlowBuilder/index.tsx:194]: Is this props required?

**Review by laurentlp — COMMENTED** [review]
Small comments other than that tested and LGTM!

File: packages/studio-ui/src/web/views/FlowBuilder/index.tsx (hunk 1 above)
Comment (sadda11asm) [reply packages/studio-ui/src/web/views/FlowBuilder/index.tsx:194]: oh, I think I forgot to delete it after some debugging

**Review by laurentlp — APPROVED** [review]
LGTM!!

**Pull request merged**